app.py

```diff
@@ -201,6 +209,25 @@ def render_end_progress_bar(self):
                            self.bar_width, self.height//100)
         pygame.draw.rect(self.screen, Color.SCREEN_BG.value, stone_div)
 
+  def render_end_progress_labels(self):
```

Comment:
Can this be accomplished with just one function and a for loop?

Something like,
```python
def render_end_progress_labels(self):
...
side_signs = (-1, 1)
for side_sign in side_signs:
  x_offset = self.center["x"] + side_sign * self.bar_x_offset
  ...
```